YvesIraguha/personalFinanceMobile
# #171691919 CRUD Investments

#### What does this PR do?
- CRUD on Investments
#### Description of Task to be completed?
- Allow a user to view  all investments
- Allow a user to view one investment details
- Allow a user to edit an investment 
- Allow a user to delete an investment 
#### How should this be manually tested?
- Clone the repo and checkout the branch `ft-crud-investements-#171691919 `
- checkout the respective branch, and run `yarn && yarn start` to start the application command
- Follow the directions provided to open the application in the simulator or real device.
-  Ensure that your back-end is working. 
- Click on the profile avatar to access investments screen and click on investment to work on investment. 

#### What are the relevant pivotal tracker stories?
#171691919
